certbot-apache/certbot_apache/configurator.py

```diff
@@ -1705,21 +1721,39 @@ def enable_site(self, vhost):
             supported.
 
         """
-        if self.is_site_enabled(vhost.filep):
+        if vhost.enabled:
+            return
+
+        # Handle non-debian systems
+        if not self.conf("handle-sites"):
+            if not self.parser.parsed_in_original(vhost.filep):
+                # Add direct include to root conf
+                self.parser.add_include(self.parser.loc["default"], vhost.filep)
```

Comment:
We should probably set `vhost.enabled` to `True` after this to make sure the value stays up to date.

Reply:
Oh, of course. Thanks!